datadog_checks_dev/datadog_checks/dev/tooling/commands/release/show/changes.py

```diff
@@ -12,11 +12,12 @@
 
 @click.command(context_settings=CONTEXT_SETTINGS, short_help='Show all the pending PRs for a given check.')
 @click.argument('check', autocompletion=complete_valid_checks, callback=validate_check_arg)
+@click.option('--organization', '-r', default=None)
```

Comment:
Should we just default to `DataDog` and avoid the conditional later?

Reply:
Sure, it's cleaner that way! I updated it to default here, and default in `get_pr()`. I'm not sure if there's a better way (ie not have to default in both places).